zio-schema/shared/src/main/scala/zio/schema/Schema.scala

```diff
@@ -31,6 +31,8 @@ sealed trait Schema[A] {
 
   type Accessors[Lens[_, _], Prism[_, _], Traversal[_, _]]
 
+  def annotations: Chunk[Any]
+
```

Comment:
I would add a method `def annotate(annotation: Any): Schema[A]` here

Reply:
👍 Will try find some time later today to do so.